plugins/flytekit-deck-standard/setup.py

```diff
@@ -6,6 +6,15 @@
 
 plugin_requires = ["flytekit"]
 
+extras = {
```

Comment:
can you add an `all` that installs all dependencies? `pandas` is a good example of a library that [does this](https://github.com/pandas-dev/pandas/blob/main/pyproject.toml#L84).